docs/ssr.md

```diff
@@ -69,3 +69,21 @@ module.exports = {
   plugins: [...otherGatsbyPlugins, 'gatsby-plugin-emotion']
 }
 ```
+
+## Puppeteer
+
+If you are using Puppeteer to prerender your application, you have to disable emotion's `speedy` option to fully render all needed CSS inside the DOM.
+
+```jsx
+import { sheet } from 'emotion'
+
+import App from './App'
+
+sheet.speedy(false)
```

Comment:
Doing this won’t work if you use `css` in App or any of its dependencies, could you show it in another file and that being the first import?

Also, is there a way to detect if the page is being prerendered with puppeteer and if so, could you make this so it only disables speedy when it’s being prerendered?